src/CMakeLists.txt

```diff
@@ -2,6 +2,14 @@ set(CMAKE_CXX_STANDARD 11)
 cmake_minimum_required(VERSION 3.16) # Required by TFL
 project(PosturePerfection VERSION 0.1.0)
 
+
+set(LIBSRC
+  doubler.cpp
```

Comment:
Should we give this a more meaningful name or even just remove this?

Reply:
Doubler has been removed. I forgot to do that so thanks for spotting!